### PR TITLE
fix(cli): fix setting flag to false

### LINKED ISSUE #110: Boolean command line options are broken
The help says there is the `--gecko` option, `true` by default, but I can't find how to set it to `false`. I tried "--gecko=false", "--gecko false", "--no-gecko", "--gecko=0". Nothing works.

The other boolean command line options are broken as well. It's impossible to pass false to them.

### PR DESCRIPTION
**Pull request by cnishina**
- This fixes `webdriver-manager update --gecko=false`
- This does not fix `webdriver-manager update --gecko=0`. Minimist interprets 0 as true.

closes #110

### CODE REVIEW

**Comment by cnishina** [comment]
Since `getValue_` returns 3 different types, the getBoolean check requires a little more verification.

File: lib/cli/options.ts [hunk 1]
```diff
@@ -31,22 +31,33 @@ export class Option {
 
   getNumber(): number {
     let value = this.getValue_();
-    if (value && typeof value === 'number') {
-      return +value;
+    if (value) {
```
Comment (sjelin) [inline lib/cli/options.ts]: what if `value` is `0`?

Comment (cnishina) [inline lib/cli/options.ts]: Good catch. My fix is for strings or numbers to return the `+value`.  For booleans, I plan to return `null`.

**Comment by cnishina** [comment]
I agree. I'll put tests together.

**Comment by cnishina** [comment]
Done.

**Pull request merged**